kombu/transport/redis.py

```diff
@@ -1066,14 +1066,17 @@ class SentinelChannel(Channel):
     from_transport_options = Channel.from_transport_options + (
         'master_name',
         'min_other_sentinels',
-        'sentinel_kwargs')
+        'sentinel_kwargs',
+        'connection_kwargs')
 
     connection_class = sentinel.SentinelManagedConnection if sentinel else None
 
     def _sentinel_managed_pool(self, async=False):
         connparams = self._connparams(async)
 
-        additional_params = connparams.copy()
+        additional_params = getattr(self,
+                                    'connection_kwargs',
+                                    connparams.copy())
```

Comment:
You do not need to update the dictionary if the attribute exists. The attribute will contain all the necessary additional settings for the connection. And if it is not, then default parameters that are contained in connparams.copy () will be used. I think this is more logical.

'BROKER_TRANSPORT_OPTIONS': {
                                             'Master_name': 'mymaster',
                                             'Connection_kwargs': {'db': 10, 'password': 'secret'}
                                             },

'BROKER_TRANSPORT_OPTIONS': {
                                             'Master_name': 'mymaster',
                                             'Connection_kwargs': {'db': 10, 'password': 'secret'}
                                             },